bundle/manifests/kuadrant.io_dnspolicies.yaml

```diff
@@ -156,6 +156,15 @@ spec:
                         type: integer
                     type: object
                 type: object
+              routingStrategy:
+                default: loadbalanced
+                enum:
+                - simple
+                - loadbalanced
+                type: string
+                x-kubernetes-validations:
+                - message: RoutingStrategy is immutable
```

Comment:
I assume we are making this immutable because it is currently not possible to move between the two strategies? Is that something we could support at somepoint? I worry that it means deleting and removing your DNS to move between the two options

Reply:
This could be me being overly cautious. It might not actually be an issue going between record types (A vs CNAME) for the target host. Let me double check that.

Reply:
I removed the validation as it should actually be possible to do this. I have created a follow on issue to look at updating the AWS provider which will currently error out when trying to update an existing record with a change of type (CNAME vs A).

https://github.com/Kuadrant/multicluster-gateway-controller/issues/664

All provider errors are already put into the DNSPOlicy ready condition so it will visible there.

Google _should_ work, but i didn't try it.